File: js/getAllResults.js
$(document).ready(function(){
			 
			 
			 $.post( "cgi-bin/getAllResults.py", function( data ) {
			     
				var innerhtml = "";
				var i;
				 $('#tableResults').html();
				 for(i=0;i<data.length;i++)
					 {
					 
					 	innerhtml +='<tr><td>'+data[i].id+'</td><td>'+data[i].ticketId+'</td><td>'+data[i].phoneNumber+'</td><td>'+data[i].registrationNumber+'</td><td>'+data[i].inTime+'</td><td>'+data[i].outTime+'</td>';
					 }
				
				 $('#tableResults').html(innerhtml);
				 		
			   
			 
			 
			},"json");
		});

File: WEB-INF/cgi/getAllResults.py
#!/usr/bin/python

import MySQLdb
import cgi

print "Content-type: application/json\n";

db = MySQLdb.connect("localhost","gate","Saradhi@2","gate2016" )


cursor = db.cursor()

sql = "SELECT * FROM transaction_tbl"

cursor.execute(sql)

results = cursor.fetchall()
rows = cursor.rowcount

count = 0;
json = "["
for row in results:
    json += '{"id":%d ,' % (row[0])
    json +=' "ticketId":"%s",' %(row[1])
    json +=' "phoneNumber":"%s",' %(row[2])
    json +='"registrationNumber":"%s",' %(row[3])
    json +='"inTime":"%s",' %(row[4])
    json +='"outTime":"%s"}' %(row[5])   
    count = count+1;
    if count != rows:
        json += ','
json +=']'
print json


db.close()